sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/cryptography/LocalCryptographyClientBuilder.java

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.security.keyvault.keys.cryptography;
+
+import com.azure.core.util.logging.ClientLogger;
+import com.azure.security.keyvault.keys.models.JsonWebKey;
+
+/**
+ * This class provides a fluent builder API to help aid the configuration and instantiation of the {@link
+ * LocalCryptographyAsyncClient cryptography async client} and {@link LocalCryptographyClient cryptography sync client},
+ * by calling {@link LocalCryptographyClientBuilder#buildAsyncClient() buildAsyncClient} and {@link
+ * LocalCryptographyClientBuilder#buildClient() buildClient} respectively
+ * It constructs an instance of the desired client.
+ *
+ * <p> The minimal configuration options required by {@link LocalCryptographyClientBuilder cryptographyClientBuilder} to
+ * build {@link LocalCryptographyAsyncClient} is ({@link JsonWebKey jsonWebKey}.
```

Comment:
Does the sync client require more configurations? If not, the document should include both async and sync client.